<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Runs::services::ResultsExporterWidget</class>
 <widget class="QWidget" name="Runs::services::ResultsExporterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>275</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Results Exporter</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0" colspan="2">
    <widget class="QLineEdit" name="edWhenFinishedRunCmd"/>
   </item>
   <item row="6" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btExportResults">
        <property name="text">
         <string>Export results</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="edExportInterval">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="suffix">
      <string> sec</string>
     </property>
     <property name="maximum">
      <number>600</number>
     </property>
     <property name="singleStep">
      <number>15</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Export interval</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="edExportDir"/>
     </item>
     <item>
      <widget class="QPushButton" name="btChooseExportDir">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Export dir</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="lstOutputFormat"/>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>When finished, run cmd</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Output format</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>CSV separator</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QRadioButton" name="rbCsvSeparatorChar">
       <property name="text">
        <string>Char:</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edCsvSeparator"/>
     </item>
     <item>
      <widget class="QRadioButton" name="rbCsvSeparatorTab">
       <property name="text">
        <string>Tabulator</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>edExportDir</tabstop>
  <tabstop>btChooseExportDir</tabstop>
  <tabstop>edExportInterval</tabstop>
  <tabstop>lstOutputFormat</tabstop>
  <tabstop>edWhenFinishedRunCmd</tabstop>
  <tabstop>rbCsvSeparatorChar</tabstop>
  <tabstop>edCsvSeparator</tabstop>
  <tabstop>rbCsvSeparatorTab</tabstop>
  <tabstop>btExportResults</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
